modules/webbrowser/src/eventhandler.cpp

```diff
@@ -438,6 +463,23 @@ CefMouseEvent EventHandler::mouseEvent(KeyModifier mods) {
     return event;
 }
 
+#ifdef WIN32
+CefTouchEvent EventHandler::touchEvent(const TouchInput& input,
+    const cef_touch_event_type_t eventType) const
+{
+    const glm::vec2 windowPos = input.currentWindowCoordinates();
+    CefTouchEvent event = {};
+    event.id = static_cast<int>(input.fingerId);
+    event.x = windowPos.x;
+    event.y = windowPos.y;
+    event.type = eventType;
+    //TODO: We should probably use key mods for touch as well:
```

Comment:
Even if they are not used right now wouldnt it make more sense to directly add them? Seems like all the mouseEvents mostly do not use them either but having KeyModifier mods = KeyModifier::NoModifier as default parameter and then event.modifiers |= static_cast<uint32_t>(mapToCefModifiers(mods)); in case someone later on wants to add functionality around it?

Reply:
Yeah this sounds like a good suggestion